src/guiguts/highlight.py

```diff
@@ -174,6 +164,13 @@ def _highlight_configure_tag(
         tag_colors: Dictionary of fg/bg colors for each theme.
     """
     theme = preferences.get(PrefKey.THEME_NAME)
+
+    if theme == "Default":
+        if maintext().is_dark_theme():
```

Comment:
Do you think it would be better/simpler to depend on `is_dark_theme` whatever the theme name, rather than only when the theme is Default, i.e. completely ignore the theme name here? The only case that would be altered is if the "Dark" theme were actually light in color, or the "Light" theme actually dark. In those (unlikely) cases, perhaps we'd prefer the colors to be chosen on the basis of the actual theme darkness, rather than the bizarre name.

Reply:
Good idea. The latest commit has that (and the code is simpler to boot).